recipes/bun/recipe.yaml

```
@@ -0,0 +1,64 @@
+context:
+  version: 1.2.13
+
+package:
+  name: bun
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/oven-sh/bun/archive/refs/tags/bun-v${{ version }}.tar.gz
+    sha256: 6aac998038ebf878a4df167e906e8c84797e83c27ac5b40c20153aa52f127080
+
+build:
+  number: 0
+  script:
+    - if: unix
+      then:
+        - export CI=OFF
+        - curl -fsSL https://bun.sh/install | bash
+        - $HOME/.bun/bin/bun run build:release
+        - mkdir -p $PREFIX/bin
+        - mv ./build/release/bun $PREFIX/bin
+    - if: win
+      then:
+        - irm bun.sh/install.ps1 | iex
+        - bun run build:release
+        - mkdir "%LIBRARY_BIN%"
+        - move build\release\bun.exe "%LIBRARY_BIN%"
+
+requirements:
+  build:
+    - lsb-release
+    - ccache
+    - cmake >=3.28
+    - git
+    - libtool
+    - ninja
+    - pkg-config
+    - ruby
+    - xz
+    - unzip
+    - lld
+    - llvm
+    - llvm-tools
+    - clang 19.*
+    - clangxx 19.*
+    - clang_impl_${{ build_platform }}
+    - clangxx_impl_${{ build_platform }}
+    - zig
+    - ${{ compiler('rust') }}
+    - ${{ compiler('go') }}
```

Please use either `go-cgo` or `go-nocgo`
```suggestion
    - ${{ compiler('go-nocgo') }}
```